pkg/destroy/gcp/instance.go

```diff
@@ -26,12 +26,12 @@ func (o *ClusterUninstaller) getInstanceNameAndZone(instanceURL string) (string,
 }
 
 func (o *ClusterUninstaller) listInstances() ([]cloudResource, error) {
-	byName, err := o.listInstancesWithFilter("items/*/instances(name,zone,status,machineType),nextPageToken", o.clusterIDFilter(), nil)
+	byName, err := o.listInstancesWithFilter("items/*/instances(labels,name,zone,status,machineType),nextPageToken", o.clusterIDFilter(), nil)
```

Comment:
I feel like I'm missing something but why do we need the labels?

Reply:
technically that is another way to identify Disks if created with the CSI. I may have left that in there as I was adding that to the return type but we can probably get rid of that if I am no longer using it as part of CloudResource objects that are returned

Reply:
Maybe this slipped in from #5976?

Reply:
squashed